.github/workflows/validate_relaxng.yml

```diff
@@ -0,0 +1,22 @@
+name: validate relaxng
+
+on: [push]
+
+jobs:
+  validate-relaxng:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@main
+    - name: Count articles
+      run: echo '::set-output name=TOTAL::`ls articles/[0-9]*/*.xml | wc -l`'
+      id: article-count
+    - name: Output article count
+      run: echo "Validating ${{ steps.article-count.outputs.TOTAL }} articles" >> GITHUB_STEP_SUMMARY
```

Comment:
The GITHUB_STEP_SUMMARY is just so that it is outputted on the Github Actions page when it is run. There is some documentation [here](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#adding-a-job-summary) for that.

Reply:
Right. That page makes it _very_ clear that the `GITHUB_STEP_SUMMARY` on line 14 should be `$GITHUB_STEP_SUMMARY`.